iocp.py

```diff
@@ -35,12 +35,16 @@
 #
 ###################################################################################################
 
+#from __future__ import unicode_literals
```

Comment:
Remnants of a change that was not kept. This line has no real purpose any longer and should not be merged.